test/enhanced_mysql_tasks/schema_statements_test.rb

```diff
@@ -14,8 +14,6 @@ def test_structure_dump
     assert_equal "INSERT INTO schema_migrations (version) VALUES
 ('20160101010101'),
 ('20160201010102'),
-('20160301010103');
-
-", ActiveRecord::Base.connection.dump_schema_information
+('20160301010103');", ActiveRecord::Base.connection.dump_schema_information
```

Comment:
Removed two line breaks since https://github.com/ppworks/enhanced_mysql_tasks/commit/6db6c6cc8ce3771db5e9c7f1d9100924c8a231e1 did not update this test.